PL2/PL2-parser/src/test/java/nl/tudelft/pl2016gr2/parser/controller/AnnotationReaderTest.java

```diff
@@ -0,0 +1,47 @@
+package nl.tudelft.pl2016gr2.parser.controller;
+
+import static org.junit.Assert.assertEquals;
+
+import nl.tudelft.pl2016gr2.model.Annotation;
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class AnnotationReaderTest {
+
+  private File tempFile;
+
+  /**
+   * Creates a reader for me specifically.
+   *
+   * @throws IOException when tempfile couldn't be created..
+   */
+  @Before
+  public void setup() throws IOException {
+    tempFile = File.createTempFile(getClass().getName(), "tempmetadatafile");
+  }
+
+  @After
+  public void teardown() {
+    tempFile.delete();
+  }
+
+  @Test
+  public void testRead() throws Exception {
+
+    FileUtils.copyInputStreamToFile(
```

Comment:
Why not use the `.xlsx` file directly?

Should also specify any fixtures (such as file names) in constants.